src/components/TransitionStaggered/README.md

```diff
@@ -0,0 +1,98 @@
+# Transition Staggered
+
```

Comment:
the example is pretty clear, but i would maybe write a couple sentences here at the top, something like "use MTransitionStaggered when you wanna animate a group of items (typically in a list or a grid) into view after lazy-loading them. the MTransitionStaggered components **has to be** wrapped around every item in the grid/list and it will know how to animate the item given the `item-index` prop"